<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>461</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Konwerter temperatury</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnFtoC">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>250</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt; °F na °C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCtoF">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>250</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>°C na °F &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>-273.15</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editFar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°Celcjusze</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>158</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°Faranhaity</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>43</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Plik</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuKonwertuj">
    <property name="title">
     <string>&amp;Konwertuj</string>
    </property>
    <addaction name="actionC_to_F"/>
    <addaction name="actionF_to_C"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuKonwertuj"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionC_to_F">
   <property name="text">
    <string>&amp;C na F</string>
   </property>
  </action>
  <action name="actionF_to_C">
   <property name="text">
    <string>&amp;F na C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
